networking/nw-create-custom-ingress-controller

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * networking/ingress-operator.adoc
+
+:_content-type: PROCEDURE
+[id="nw-create-custom-ingress-controller_{context}"]
+= Creating a custom Ingress Controller
+
+When creating an custom Ingress Controller, 
+
+.Prerequisites
+
+* Must have cluster-admin permissions to the cluster
+* Create a custom wildcard certificate for the custom domain.
+?Do we have a link to this elsewhere...(# oc create secret tls custom-ingress-custom-certs -n openshift-ingress --cert=<certificate.crt> --key=<key.key>)
+
+.Procedure
+
+. Define the ingress controller in a file as template:
++
+[source,yaml]
+----
+# cat custom-ingress-controller.yaml
+
+apiVersion: operator.openshift.io/v1
+kind: IngressController
+metadata:
+name: custom
+namespace: openshift-ingress-operator
+spec:
+defaultCertificate:
+    name: custom-ingress-custom-certs
+domain: "custom-domain.mycompany.com"
+nodePlacement:
+    nodeSelector:
+    matchLabels:
+        ingress-custom-controller: "true"
+replicas: 2
+status: {}
+----
+
+. Set the `spec.domain` to your domain name.
```

Comment:
This is worth stating explicitly:  The domain specified on the IngressController object and the domain used for the certificate must match.  For example, if the IngressController has `domain: custom-domain.mycompany.com`, then the certificate must have SAN `*.custom-domain.mycompany.com` (with `*.` added to the domain).

Reply:
ty that is worth stating explicitly